### ISSUE TITLE
Ambiguous id field for MSSQL and ORDER for find

### ISSUE THREAD

**Issue by jifeon** [sequelize/sequelize#4981]
sequelize 3.14.0

Site.js
```js
const Site = sequelize.define('Site', {
  id: {
    type: Sequelize.INTEGER,
    autoIncrement: true,
    primaryKey: true,
    field: 'SiteID',
  },

  uri: {
    type: Sequelize.STRING,
    field: 'Uri',
  }
});


Site.hasOne(ContactModule, {
  as: 'contact',
});
```

ContactModule.js
```js
const ContactModule = sequelize.define('ContactModule', {
  id: {
    type: Sequelize.UUID,
    primaryKey: true,
    field: 'ModuleID',
    defaultValue: Sequelize.UUIDV4,
    allowNull: false,
  },

  siteId: {
    type: Sequelize.INTEGER,
    allowNull: false,
    field: 'SiteID',
  },
});
```

query.js
```js
Site.find({
  where: {
    uri: siteUri,
  },
  include: [{model: ContactModule, as: 'contact'}],
});
```

It adds ORDER BY to the query, which cannot be redefined:

```sql
WHERE [Site].[Uri] = 'test'
ORDER BY [SiteID] OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY;
```

`Ambiguous column name 'SiteID'.`

SiteID is ambiguous because both Site and ContactModule has a field with such name.

**Comment by jifeon** [comment]
maybe related to #2496

**Comment by AckerApple** [comment]
I'm going to try my dern' best right now to make my first pull request, to fix for this. It's an easy fix:

At Line:692 of lib/dialects/mssql/query-generator.js INSERT THIS CODE:
```
if(model.name){
  fragment += this.quoteIdentifier(model.name) + '.';
}
```

Wish me luck!

**Comment by AckerApple** [comment]
I made the pull request to fix

https://github.com/sequelize/sequelize/pull/6181

**Comment by sushantdhiman** [comment]
Fixed by #6181
